tests/template_tests/templatetags/custom.py

```diff
@@ -51,6 +56,12 @@ def one_param(arg):
 one_param.anything = "Expected one_param __dict__"
 
 
+@register.simple_block_tag
+def oneparamblock(content, arg):
```

Comment:
It's consistent with the cases for `simple_tag`, but not with the naming of built-in tags. Personally, I prefer the look of tags without underscores in, but happy to change them if that's the better way to go.

Reply:
I see what you mean :thinking: we can leave it @nessita do you have a preference here?

Reply:
Thanks for summon me, could you please allow me for a few days to do a full review and comment?

Reply:
After a careful review, let's **please** add underscores in these new tags since that's the style of the custom.py file. Otherwise the inconsistency kind of hurts my eyes :rofl: [MY EYES, MY EYES](https://tenor.com/view/my-eyes-friends-phoebe-phoebe-buffay-meme-gif-25833919)!

Reply:
It works on my ~machine~ eyes :wink: 

(Makes sense - I'll make the change).